source/usb/cdc/usbd_core_cdc.c

```diff
@@ -93,9 +94,18 @@ __weak BOOL USBD_EndPoint0_Setup_CDC_ReqToIF(void)
                 break;
 
             case CDC_SEND_BREAK:
+								if (!dap_lock_operation(DAP_LOCK_OPERATION_CDC_BREAK))
+								{
+									util_assert(0);
```

Comment:
Asserts should only trigger when a DAPLink bug occurs. This is asserting on user error - break is sent while something else is using SWD.